andromedocs-theme/src/components/header.tsx

```diff
@@ -7,7 +7,7 @@ const Header = () => (
     <Flex
       sx={{
         height: "80px",
-        backgroundColor: "muted.5",
+        backgroundColor: "muted.6",
```

Comment:
The `muted.6` value doesn't exist, I think :eyes: Shouldn't this remain as `muted.5` (which is white)?